### ISSUE TITLE
dotnet new -l -tag Web

### ISSUE THREAD

**Issue by doggy8088** [dotnet/templating#3197]
There is an option for filtering by language like below which can filter templates by `C#`:

```sh
dotnet new -l -lang C#
```

I'd like to suggest a new argument `-t` for filtering tags.  For example:

```sh 
dotnet new -l -tag Web
dotnet new -l -tag MVC
dotnet new -l -tag MVC,xUnit
```

**Comment by bekir-ozturk** [comment]
Hi @doggy8088,
 Tag filter was delivered with #2475, used as `--tag`. Closing this one.

**Comment by doggy8088** [comment]
Is this option only available on .NET 6?

**Comment by vlada-shubina** [comment]
This option is available from SDK 5.0.3xx. Thanks
